rotkehlchen/tests/fixtures/history.py

```diff
@@ -24,6 +25,16 @@ def fixture_defillama():
     return Defillama()
 
 
+@pytest.fixture(name='uniswapv2')
+def fixture_uniswapv2(ethereum_inquirer):
```

Comment:
same for the other and the function name
```suggestion
@pytest.fixture(name='uniswapv2_inquirer')
def fixture_uniswapv2(ethereum_inquirer):
```